kubernetes/loculus/values.yaml

```diff
@@ -25,6 +25,11 @@ dataUseTerms:
   urls:
     open: https://#TODO-MVP/open
     restricted: https://#TODO-MVP/restricted
+sequenceFlagging:
```

Comment:
Do we want to have an `enabled: boolean` again?

Reply:
I don't think that in this case that would be helpful. If the setting is there, it's enabled. if it isn't there it is disabled, I think that makes sense

Reply:
In the helm charts I've seen, there usually is a simple boolean flag even if there's more complex config - probably to allow disabling at the flip of a boolean, as opposed to the addition/deletion of a lot of yaml.